testdata/server.key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
🛑 **Gitleaks** has detected a secret with rule-id `private-key` in commit 4a487e42b42d6d609a558675af0ccc04aedb8bc1.
If this secret is a _true_ positive, please rotate the secret ASAP.

If this secret is a _false_ positive, you can add the fingerprint below to your `.gitleaksignore` file and commit the change to this branch.

```
echo 4a487e42b42d6d609a558675af0ccc04aedb8bc1:testdata/server.key:private-key:1 >> .gitleaksignore
```